### PR TITLE
fix(kit): add missing dependency `@ng-web-apis/resize-observer`

### LINKED ISSUE #3982: 🐞 - New dependency that breaks build in 3.21(at least)
### Playground Link

https://stackblitz.com/run?file=src%2Fapp%2Fapp.component.html&hideExplorer=1

### Description

After upgrading from taiga-ui 3.15 to 3.21 there is new dependency that breaks the build
for example you can check this repository
https://github.com/Alexander-Panimash/test-taiga-ui/tree/master/test-taiga-ui
image

### Angular version

14, 15

### Taiga UI version

3.21

### Which browsers have you used?

- [ ] Chrome
- [ ] Firefox
- [ ] Safari
- [ ] Edge

### Which operating systems have you used?

- [ ] macOS
- [ ] Windows
- [ ] Linux
- [ ] iOS
- [ ] Android

### PR DESCRIPTION
**Pull request by vladimirpotekhin**
## PR Checklist

Please check if your PR fulfills the following requirements:

- [x] The commit message follows [Conventional Commits](https://www.conventionalcommits.org/en/)
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

- [x] Bugfix
- [ ] Feature
- [ ] Refactoring
- [ ] Code style update
- [ ] Build or CI related changes
- [ ] Documentation content changes

## What is the current behavior?

Closes #3982
## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

**Review by splincode — APPROVED** [review]

File: projects/kit/package.json [hunk 1]
```diff
@@ -14,6 +14,8 @@
     "license": "Apache-2.0",
     "dependencies": {
         "@ng-web-apis/intersection-observer": "3.0.0",
+        "@ng-web-apis/mutation-observer": "2.0.0",
```
Comment (nsbarsukov) [inline projects/kit/package.json:17]: It is already inside `peerDependencies`

Comment (waterplea) [reply projects/kit/package.json:17]: We shouldn't add them as dependencies. They should be in peer dependencies. The issue is `@ng-web-apis/resize-observer` is missing there.

Comment (vladimirpotekhin) [reply projects/kit/package.json]: u r right, thanks

File: projects/kit/package.json [hunk 2]
```diff
@@ -23,6 +23,7 @@
         "@angular/router": ">=12.0.0",
         "@ng-web-apis/common": ">=2.0.0",
         "@ng-web-apis/mutation-observer": ">=2.0.0",
+        "@ng-web-apis/resize-observer": "2.0.0",
```
Comment (splincode) [inline projects/kit/package.json:26]: `">=2.0.0"`

**Review by nsbarsukov — APPROVED** [review]

**Pull request merged**